### DISCUSSION TITLE
Looking for maintainers

### DISCUSSION THREAD

**Discussion by baetheus** [nullpub/datum#20] [Q&A]
Hello All!

I'm opening up discussions to primarily look for people who are interested in helping me maintain datum! I have no plans to step down but I'd like to invite others to help manage datum. My day to day work is ramping up and I'm getting closer and closer to the limits of what I can learn from TypeScript (as in I'm going to start delving into other languages).

So! I guess I should start with where I see datum going:

1. @SamProtas has graciously helped a lot recently to get datum more inline with where fp-ts is heading. The 3.5 release is pretty much all his work and includes a new data type (DatumThese) as well as module instances for Datum. This rounds out my initial vision for the entire repo.
2. I'd like to target [the 4.0 release](https://github.com/nullpub/datum/projects/1) with some code cleanup, bug fixes, and removals to complete the work that he has started.
3. Aside from DatumThese I don't think there is much more that is needed in datum. The only other feature I had on the horizon is a concept of progress, which has been languishing for some time and can be hit in the distant future.
4. General use documentation (with examples) is much needed.
5. Someone other than me joining as a maintainer with reviewer and merge access.

I'm asking for help with any and all of the above. I'm also more than willing to guide anyone that wants to help out but doesn't think they have the skills.

Anyway, thanks for reading and I hope to hear from you!

**Comment by SamProtas** [comment]
Hey @baetheus I've been temporarily pulled away from some of this work, but I do plan to return to the 4.0 release we talked about (including less glamorous but important things like docs/examples). I'm happy to help review/merge Issues/PRs from others as well.

**Reply by baetheus** [reply]
Awesome! I had given you Write access before but I've upped that to Maintainer. I'll likely add you to the npm contributors list at some point too. Thanks for stepping up.
